src/ci/docker/host-x86_64/x86_64-gnu-llvm-13/Dockerfile

```diff
@@ -40,6 +40,7 @@ RUN sh /scripts/sccache.sh
 # We are disabling CI LLVM since this builder is intentionally using a host
 # LLVM, rather than the typical src/llvm-project LLVM.
 ENV NO_DOWNLOAD_CI_LLVM 1
+ENV EXTERNAL_LLVM 1
```

Comment:
Note that the source could be downloaded with `apt-get source llvm-13`, so we should be able to provide the current source for this builder.

Reply:
I don't think that's a good idea, see https://github.com/rust-lang/rust/pull/102579#discussion_r1032930992.